packages/Fable.Form.Simple.Bulma/Form.fs

```diff
@@ -465,6 +470,18 @@ module Form =
                                     if state = Loading then
                                         button.isLoading
                                 ]
+                                match cancelPolicy with
+                                    | CancelPolicy.DoNothing -> Html.none
+                                    | CancelPolicy.Action action ->
+                                        Bulma.button.button [
+                                            color.isLight
+                                            prop.text "Cancel"
```

Comment:
The text should be configurable because on some forms the "cancel policy" could be to reset the form values.